pyproject.toml

```diff
@@ -0,0 +1,21 @@
+[tool.poetry]
+name = "core-ct"
+version = "0.1.0"
+description = "A Python library to assist geologists with the analysis of rock core CT scans"
+authors = ["Your Name <you@example.com>"]
+license = "MIT"
+readme = "README.md"
+
+[tool.poetry.dependencies]
+python = ">=3.11,<3.13"
```

Comment:
How did you decide on the range 3.11-3.13 for Python version? I use WSL with Ubuntu and for some reason Python 3.11 is hard to get to work with WSL. I ran it with Python 3.10 as the lower bound instead and it seemed to work for me, but I suppose that doesn't mean 3.10 is guaranteed to work for everyone. I think we should consider changing the lower bound to 3.10 though.

Reply:
TLDR: There isn't any good reason for the lower bound to stay at 3.11 so I went ahead and lowered it to 3.10.

Since I have Python 3.11 installed, when I configured the repository using `poetry` it automatically assigned the version to Python 3.11. The reason for the `<3.13` is due to the fact that Numpy doesn't work with Python 3.13 so I needed to place a cap on it. The 3.13 needs to stay in order for `poetry` to manage `numpy` correctly, but I went ahead and made the lower bound Python 3.10